backend/models/request_models.py

```diff
@@ -6,3 +6,11 @@
 class InsertDataRequest(BaseModel):
     table_name: str
     values: Dict[str, Any]
+
+
+class QueryRequestModel(BaseModel):
+    user_input: str
+
+
+class QueryResponseModel(BaseModel):
```

Comment:
This file is named requestmodels, I think it would be a good idea to make a file named response_models